UnitTests/TableViewTests.cs

```diff
@@ -778,6 +778,85 @@ public void ScrollRight_WithoutSmoothScrolling ()
 			Application.Shutdown ();
 		}
 
+
+		[Fact]
+		public void ScrollIndicators ()
+		{
+			GraphViewTests.InitFakeDriver ();
+
+			var tableView = new TableView ();
+			tableView.ColorScheme = Colors.TopLevel;
+
+			// 3 columns are visibile
+			tableView.Bounds = new Rect (0, 0, 7, 5);
+			tableView.Style.ShowHorizontalHeaderUnderline = true;
+			tableView.Style.ShowHorizontalHeaderOverline = false;
+			tableView.Style.AlwaysShowHeaders = true;
+			tableView.Style.SmoothHorizontalScrolling = true;
+
+			var dt = new DataTable ();
+			dt.Columns.Add ("A");
+			dt.Columns.Add ("B");
+			dt.Columns.Add ("C");
+			dt.Columns.Add ("D");
+			dt.Columns.Add ("E");
+			dt.Columns.Add ("F");
+
+			dt.Rows.Add (1, 2, 3, 4, 5, 6);
+
+			tableView.Table = dt;
+
+			// select last visible column
+			tableView.SelectedColumn = 2; // column C
+
+			tableView.Redraw (tableView.Bounds);
+
+			// user can only scroll right so sees right indicator
+			// Because first column in table is A
+			string expected = 
+				@"
+│A│B│C│
```

Comment:
Just curious: How exhaustive are the unit tests for tableview with wide runes?

Reply:
I don't think there are any tests covering it.  But `CalculateMaxCellWidth` does use `col.ColumnName.Sum (c => Rune.ColumnWidth (c));` so it should work.  

I am happy to write some though.  Always good to have more tests.

Reply:
@tig  I have added a PR #1761 that should ensure any unicode issues are easily diagnosable